test/prob/test_fixture_ccdf_log.hpp

```diff
@@ -203,26 +206,26 @@ class AgradCcdfLogTestFixture : public ::testing::Test {
 
   void add_finite_diff_1storder(const vector<double>& params,
                                 vector<double>& finite_diff, const size_t n) {
-    const double e = 1e-8;
-    const double e2 = 2 * e;
-
-    vector<double> plus(6);
-    vector<double> minus(6);
-    for (size_t i = 0; i < 6; i++) {
-      plus[i] = get_param<double>(params, i);
-      minus[i] = get_param<double>(params, i);
-    }
-    plus[n] += e;
-    minus[n] -= e;
+    auto f_wrap = [&](const Eigen::VectorXd& e) {
```

Comment:
The stan math finite difference function is higher order and easier to use, so I defer to it.